app/pandas.ipynb

```diff
@@ -549,14 +592,19 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
-   "outputs": [],
    "source": [
-    "# write your code here\n"
+    "large_s = pd.Series(np.random.randint(1, 101, size=10000000))\n",
+    "average_s = large_s.rolling(window=4, min_periods=1).mean()"
```

Comment:
The rolling window size is set to 4, but the task requires a window of 5 elements. Change `window=4` to `window=5` to correctly calculate the moving average over 5 elements.